source/less/style.less

```
@@ -1 +1,20 @@
-// Начните писать стили здесь
+@import "./global/variables.less";
+@import "./global/fonts.less";
+@import "./global/mixins.less";
+@import "./global/global.less";
+@import "./global/visually-hidden.less";
+@import "./components/page-header.less";
```

если я не ошибаюсь, можно писать без разширения файла:
```suggestion
@import "./components/page-header";
```